src/Form/CitizenProjectCommandType.php

```diff
@@ -55,10 +65,38 @@ public function buildForm(FormBuilderInterface $builder, array $options)
             ])
             ->addEventListener(
                 FormEvents::PRE_SET_DATA, [$this, 'preSetData']
-            )
-        ;
+            );
 
         $builder->get('address')->remove('address');
+
+        /** @var CitizenProjectCommand $citizenProjectCommand */
+        $citizenProjectCommand = $builder->getData();
+
+        if (
+            $citizenProjectCommand instanceof CitizenProjectCreationCommand
+            || ($citizenProjectCommand->getCitizenProject() && $citizenProjectCommand->getCitizenProject()->isPending())
```

Comment:
The `||` will break the second statement if the first is false.

Reply:
✔️